gm4_combat_expanded/data/gm4_combat_expanded/functions/armor/modifier/type/lunar/check.mcfunction

```diff
@@ -0,0 +1,17 @@
+# check moon phase and time of day
+# @s = player wearing armour
+# run from armor/check_modifier
+
+# get moon cycle (0 = full moon)
```

Comment:
I think the method I'm using is better, the storing of the phase in the item nbt is so I can check if it still matches (since armor can be taken off for multiple nights), without that I don't see a way to check if the armor should update.

The scoreboard method seems better than having 5 predicates to check the different phases